platform/ext/target/nordic_nrf/common/nrf_common.cmake

```diff
@@ -147,3 +147,7 @@ elseif(BUILD_FLASH)
   set(ITS_CREATE_FLASH_LAYOUT ON)
   embedded_include_directories(PATH "${NRF_COMMON_DIR}/cmsis_drivers" ABSOLUTE)
 endif()
+
+if(PSA_API_TEST_NS)
+  embedded_set_target_link_defines(TARGET tfm_s DEFINES "PSA_API_TEST_NS")
```

Comment:
Could you please add a comment here explaining why this is needeD?

Reply:
```suggestion
  # We want to place the veneers in different locations depending on this define.
  embedded_set_target_link_defines(TARGET tfm_s DEFINES "PSA_API_TEST_NS")
```